frontend/src/layout/nav.vue

```diff
@@ -88,7 +88,8 @@
       isGAorCSR() {
         if (this.user && this.user.role) {
           let { role_code } = this.user.role
-          if (role_code === 'CSR' || role_code === 'GA' || role_code === 'LIAISON' ) {
+          let { liaison_designate } = this.user.liaison_designate
+          if (role_code === 'CSR' || role_code === 'GA' ||  liaison_designate === 1 ) {
```

Comment:
I don't think we need to check for liaison_designate here anymore - they can be any number of things depending on their role.

Reply:
Fixed.